.github/workflows/build-docker.yml

```diff
@@ -11,6 +11,9 @@ on:
       options:
         type: string
         default: -DENABLE_CUDA=off
+    secrets:
+      DOCKERHUB_PASSWORD:
```

Comment:
I thought that secrets are global vars for all github jobs, but after discussion with Alexei I understood that it's special github behavior https://github.blog/changelog/2022-05-03-github-actions-simplify-using-secrets-with-reusable-workflows/ . Maybe there are some WA, because secrets added as https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-an-environment . 

This is only to mark and summarize our discussion.